src/components/Table/ResourceAvailabilityTable.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="text-center">
-    <h1>{{ user.name }}'s resource table</h1>
+    <h1>{{ user.firstName }}'s resource table</h1>
```

Comment:
First Name Last name would be better here.

Reply:
done